nextcord/client.py

```diff
@@ -2045,7 +2056,7 @@ def slash_command(
         description: :class:`str`
             Description of the command that users will see. If not set, it defaults to the bare minimum Discord allows.
         guild_ids: Iterable[:class:`int`]
-            IDs of :class:`Guild`'s to add this command to. If unset, this will be a global command.
+            IDs of :class:`Guild`'s to add this command to. If unset, it will use the client rollout guild ids.
```

Comment:
```suggestion
            IDs of :class:`Guild`'s to add this command to. If unset, ``rollout_guild_ids`` set on the
            :class:`~.ext.commands.Bot` or :class:`Client` will be used. If neither are present, this will be a global command.
```